okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -1050,6 +1035,99 @@ private ModuleDescriptor findSystemInterface(Tenant tenant, String interfaceName
     return promise.future();
   }
 
+  private boolean isExclusive(ModuleDescriptor md) {
+    return md.getSystemInterface("_tenantPermissions") != null
+        || md.getAuthRoutingEntry() != null;
+  }
+
+  private boolean depsOK(TenantModuleDescriptor tm, ModuleDescriptor md,
+      Collection<ModuleDescriptor> modules, Set<String> allProvided) {
+
+    if (tm.getAction().equals(Action.disable)) {
```

Comment:
Can you replace `.equals` by `==` when comparing enums for readability, better type check, and avoiding NPE?